solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -65,6 +68,8 @@ public void init(List<String> solrUrls) throws Exception {
       urlScheme = solrUrl.startsWith("https") ? "https" : "http";
       try (SolrClient initialClient = getSolrClient(solrUrl)) {
         this.liveNodes = fetchLiveNodes(initialClient);
+        this.initialNodes = Set.copyOf(liveNodes);
```

Comment:
I think the idea of this JIRA issue is that we'll take the Solr URLs as configured and use *this* is the initial / backup liveNodes.  I think this is a very simple idea to to document/understand/implement.